File: 12403 Save Setu/solution.c
#include <stdio.h>
#include <string.h>

int main() {
    int T;
    scanf("%d", &T);
    char operation[7];
    int total = 0;
    while(T--) {
        scanf("%s", operation);
        if(strcmp(operation, "donate") == 0) {
            int K;
            scanf("%d", &K);
            total += K;
        } else
            printf("%d\n", total);
    }
	return 0;
}

File: 12403 Save Setu/solution.java
import java.io.*;

public class Main {
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		StreamTokenizer st = new StreamTokenizer(br);
		st.nextToken();
		int T = (int)st.nval, total = 0;
		StringBuilder output = new StringBuilder();
		while(T-- > 0) {
		    st.nextToken();
		    if(st.sval.equals("donate")) {
		        st.nextToken();
		        total += (int)st.nval;
		    } else
		        output.append(total).append('\n');
		}
		System.out.print(output);
	}
}
